src/main/java/de/dhbw/karlsruhe/students/mailflow/core/domain/email/entities/EmailMetadata.java

```diff
@@ -0,0 +1,68 @@
+package de.dhbw.karlsruhe.students.mailflow.core.domain.email.entities;
+
+import de.dhbw.karlsruhe.students.mailflow.core.domain.common.models.Entity;
+import de.dhbw.karlsruhe.students.mailflow.core.domain.email.value_objects.Address;
+import de.dhbw.karlsruhe.students.mailflow.core.domain.email.value_objects.EmailMetadataId;
+import de.dhbw.karlsruhe.students.mailflow.core.domain.email.value_objects.Recipients;
+import de.dhbw.karlsruhe.students.mailflow.core.domain.email.value_objects.Subject;
+import de.dhbw.karlsruhe.students.mailflow.core.domain.email.value_objects.SentDate;
+
+/**
+ * Representation of the e-mail metadata, consisting of the data subject,sender,
+ * recipents,sentDate and the isRead status
+ */
+public class EmailMetadata extends Entity<EmailMetadataId> {
```

Comment:
Why is this an entity and not a value object?

Reply:
In my opinion, `isRead` should go to the Email class, then.

Reply:
It's not exactly metadata if it's changing.

Reply:
> In my opinion, `isRead` should go to the Email class, then.

I'm fine with that